lib/opal/nodes/literal.rb

```diff
@@ -288,5 +288,29 @@ def compile_range_initialize
         push 'Opal.Range.$new(', expr(start), ',' , expr(finish), ', true)'
       end
     end
+
+    # 0b1111r -> s(:rational, (15/1))
+    # -0b1111r -> s(:rational, (-15/1))
+    class RationalNode < Base
+      handle :rational
+
+      children :value
+
+      def compile
+        push "Opal.Rational.$new(#{value.numerator}, #{value.denominator})"
```

Comment:
Should it be `Opal.get('Rational')` to make it compatible with `opal/mini`? Well, not compatible. It should raise a proper error when `Rational` is not defined, right?